.github/workflows/ci-fmudataio.yml

```diff
@@ -2,7 +2,7 @@ name: Test
 
 on:
   pull_request:
-    branches: [main]
+    branches: [main, staging]
```

Comment:
Tests don't technically _need_ to be run on the staging branch, but with the new logic it might be a nice catch